### DISCUSSION TITLE
CP-SAT : IPython.display visualization

### DISCUSSION THREAD

**Discussion by Nutchareessw** [google/or-tools#2879] [CP-SAT questions] [answered]
I have worked on job shop  problem and I run example of jobshop_ft06_sat.py on colab but output always print only Optimal makespan instead of showing visualization.RunFromIPython(): and visualization.DisplayJobshop. 

How the visualization.RunFromIPython() run? Any help much appreciate.

**Comment by lperron** [comment, marked as answer]
See https://github.com/google/or-tools/blob/86d4c543f717fa8716a9d66c25143660668bf825/ortools/sat/python/visualization.py#L17-L24

Make sure you have plotly and svgwrite installed

**Reply by Nutchareessw** [reply]
Thanks for your help @lperron. All good now.
